docs/releases/4.1.txt

```diff
@@ -179,10 +179,14 @@ File Uploads
 
 Forms
 ~~~~~
-
 * The new :meth:`~django.forms.BoundField.legend_tag` allows rendering field
   labels in ``<legend>`` tags via the new ``tag`` argument of
   :meth:`~django.forms.BoundField.label_tag`.
+  
+* Added :attr:`~django.forms.modelformset_factory.edit_only` and
+  :attr:`~django.forms.inlineformset_factory.edit_only` mode to the
+  :func:`~django.forms.modelformset_factory` and :func:`~django.forms.inlineformset_factory`,
```

Comment:
Sphinx is emitting warnings (to see them yourself cd into the docs directory and `make html` assuming you've first `python3 -m pip Sphinx`)

```suggestion
  :func:`~django.forms.models.modelformset_factory` and :func:`~django.forms.models.inlineformset_factory`,
```